packages/core/src/ceramic.ts

```diff
@@ -240,12 +226,11 @@ export class Ceramic implements CeramicApi {
 
     this.context = {
       api: this,
-      anchorService: modules.anchorService,
       ipfs: modules.ipfs,
       loggerProvider: modules.loggerProvider,
     }
     if (!this._gateway) {
-      this.context.anchorService.ceramic = this
+      this.anchorService.ceramic = this
```

Comment:
😢

Reply:
A necessary evil now, that we'll change someday. In prod it is a no-op though, so not as harmful as it looks.